homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -246,9 +252,15 @@ def __init__(self, config):
             domain = entity.domain
             if service == SERVICE_TURN_ON:
                 service = SERVICE_OPEN_COVER
-            else:
+            elif service == SERVICE_TURN_OFF:
```

Comment:
Without an `else` what happens when you get something other than turn on or turn off?

Reply:
fair enough. I changed it.